src/test/rpc_test_common.h

```diff
@@ -47,30 +47,42 @@
 #include <cart/types.h>
 #include <gurt/common.h>
 
+#define TEST_RPC_COMMON_BASE 0x010000000
+#define TEST_RPC_COMMON_VER  0
+
+
 enum {
 /* T1.0:- Without reply from server */
-CRT_RPC_TEST_IO =		(0x54312e30),
+CRT_RPC_TEST_IO =		CRT_PROTO_OPC(TEST_RPC_COMMON_BASE,
```

Comment:
Do we still need all these defines now?

Reply:
These operands are being used in the existing tests. Are you implying some other change which would be needed?